config/webpack.prod.config.js

```diff
@@ -9,6 +9,9 @@ const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin');
 const project = require('./project');
 
 const GLOBALS = {
+  'process.env': {
+    'NODE_ENV': JSON.stringify('production')
```

Comment:
it's a bit weird to me it's looking under `process.env.NODE_ENV`, when the next line just does `DEV`. Maybe change the next block as well?

Reply:
This project is still on webpack 3 and this is default configuration we had for other our plugins:

```
const GLOBALS = {	
  'process.env': { 'NODE_ENV': JSON.stringify('production') },	
  __DEV__: JSON.stringify(JSON.parse(process.env.DEBUG || 'false'))	
};
```

https://github.com/mongodb-js/compass-explain-plan/pull/5/files#diff-c227c0e0693922d0e5bdcc12cb932e41L10

Reply:
gotcha!